engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/KafkaAvroSourceFactory.scala

```diff
@@ -47,20 +50,33 @@ class KafkaTypedAvroSourceFactory[T: TypeInformation](schemaRegistryProvider: Sc
              )
              @NotBlank
              topic: String,
-             // json or avro schema on this level?
              @ParamName("schema")
              @NotBlank
-             avroSchema: String): Source[T] with TestDataGenerator = {
+             stringSchema: String): Source[T] with TestDataGenerator =
```

Comment:
the most important thing here is that is avro schema - not in string form...

Reply:
Changed.